pkg/services/alerting/notifiers/slack.go

```diff
@@ -100,11 +105,11 @@ func init() {
 				PropertyName: "mentionChannel",
 			},
 			{
-				Label:        "Token",
+				Label:        "URL",
 				Element:      alerting.ElementTypeInput,
 				InputType:    alerting.InputTypeText,
-				Description:  "Provide a bot token to use the Slack file.upload API (starts with \"xoxb\"). Specify Recipient for this to work",
-				PropertyName: "token",
+				Placeholder:  "Optionally provide a Slack incoming webhook URL for sending messages, in this case the token isn't necessary",
```

Comment:
Not sure if punctuation in a field placeholder is an improvement.

Reply:
Shouldn't this be on the `Description` field rather than placeholder. Looks weird looking at a configured channel:
![image](https://user-images.githubusercontent.com/1668778/115693103-e5f21400-a35f-11eb-9c0e-9cb12960fb58.png)

Reply:
Good catch @marefr